src/mobu/main.py

```diff
@@ -70,7 +140,16 @@ async def lifespan(app: FastAPI) -> AsyncIterator[None]:
 # Attach the routers.
 app.include_router(internal_router)
 app.include_router(external_router, prefix=config.path_prefix)
-app.include_router(github_router, prefix=f"{config.path_prefix}/github")
+
+if config.github_ci_app.enabled:
+    app.include_router(
+        github_ci_app_router, prefix=f"{config.path_prefix}/github/ci"
+    )
+if config.github_refresh_app.enabled:
+    app.include_router(
+        github_refresh_app_router,
+        prefix=f"{config.path_prefix}/github/refresh",
+    )
```

Comment:
Since you're conditionally adding the routers, I think you can drop the enabled checks and 404 exceptions in the handlers.